src/SFA.DAS.Roatp.CourseManagement/InnerApi/Models/RegisteredProvider/Feedback.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.Roatp.CourseManagement.InnerApi.Models.RegisteredProvider
+{
+    public class Feedback
```

Comment:
Think we should use proper name EmployerFeedback

Reply:
I have used the names that come back from roatp-service exactly, so diverging will cause us mapping and mapping back issues if we diverge, I suspect.  I'll leave it as is.